src/pages/patientView/mutation/PatientViewMutationsDataStore.ts

```diff
@@ -18,13 +18,9 @@ function mutationIdKey(m: Mutation) {
 export default class PatientViewMutationsDataStore extends SimpleGetterLazyMobXTableApplicationDataStore<
     Mutation[]
 > {
-    @observable.ref private mouseOverMutation: Readonly<Mutation> | null = null;
+    @observable mouseOverMutation: Readonly<Mutation> | null = null;
```

Comment:
did you mean to take @observable.ref off?

Reply:
```